spec/controllers/sign_in/application_controller_spec.rb

```diff
@@ -355,6 +366,127 @@ def service_account_auth
     end
   end
 
+  describe '#access_token_authenticate' do
```

Comment:
It looks like there aren't specs here when the `skip_error_handling` param is set to `true`